setup.py

```diff
@@ -18,7 +18,7 @@
         'dnspython >= 1.15.0',
         'enum34;python_version<"3.4"',
         'greenlet >= 0.3',
-        'monotonic >= 1.4',
+        'monotonic >= 1.4;python_version<"3.3"',
```

Comment:
```suggestion
        'monotonic >= 1.4;python_version<"3.5"',
```
Changed in version 3.5: The function is now always available and always system-wide.